android-client-sdk/src/main/java/com/devcycle/sdk/android/model/User.kt

```diff
@@ -19,7 +19,7 @@ import kotlin.IllegalArgumentException
 internal data class User private constructor(
     @Schema(required = true, description = "Unique id to identify the user")
     @JsonProperty("user_id")
-    val userId: String,
+    var userId: String,
```

Comment:
this should remain immutable, a user object shouldn't have its userId modified. Any change to userId should always require a new user object (this helps prevented unwanted side effects of various bits of code unexpectedly mutating the currently active user instance)